Dockerfile

```diff
@@ -14,17 +14,19 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
     --mount=type=bind,target=. \
     CGO_ENABLED=0 go build -o /bin/openfga ./cmd/openfga
 
-FROM cgr.dev/chainguard/static@sha256:676e989769aa9a5254fbfe14abb698804674b91c4d574bb33368d87930c5c472
+FROM alpine:3.18.4
```

Comment:
I'm not sure we want to change the base image. Chainguard was added here: https://github.com/openfga/openfga/pull/628

Could we rever this change and install `sh` manually?

Reply:
One of the whole points of chainguard is to avoid a shell-less environment. So that's an anti-pattern here IMO.